RedHatInsights/insights-inventory-frontend
# Stale fitler

Resolves: [ESSNTL-3914](https://issues.redhat.com/browse/ESSNTL-3914)

This removes default filtering completely while adding the 'Unknown' option into the status filter to be able to filter by all 4 options.

To test:

1. Visit the inventory app
2. observe that there is not any active filter by default
3. open the status filter, and observe that there is a new option 'Unknown'
4. play with the filtering: should persist refresh, should be added into URL, should filter as expected
